<launch>

  <param name="Pioneer3AT/urdf" command="$(find xacro)/xacro.py $(find pioneer3at)/urdf/pioneer3at.xacro" />
   
  <node pkg="robot_state_publisher" type="state_publisher" name="Pioneer3AT_tf_broadcaster"> 
    <param name="tf_prefix" type="string" value="Pioneer3AT"/>
    <param name="publish_frequency" type="double" value="20.0"/>
    
    <remap from="robot_description" to="/Pioneer3AT/urdf" />
    <remap from="joint_states" to="Pioneer3AT/joint_states"/>
  </node>
  
  <node pkg="pioneer3at" type="joint_state_publisher" name="Pioneer3AT_joints_publisher">
    <param name="tf_future_date" type="double" value="0.05" />
    
   <remap from="joint_states" to="Pioneer3AT/joint_states"/>
  </node>
  
</launch>
